static/css/legacy.less

```diff
@@ -1198,6 +1198,44 @@ form {
     input[type=email],
     input[type=password] {
       float: left;
+  &.floatform {
+    font-family: "Lucida Grande", "Trebuchet MS", Geneva, Helvetica, sans-serif;
+
+    .label {
+      width: 560px;
+      padding-top: 20px;
+      label {
+        font-size: 1.0em;
+        font-family: "Lucida Grande", "Trebuchet MS", Geneva, Helvetica, Arial, sans-serif;
+        font-weight: 700;
+      }
+      span {
+        font-weight: normal;
+      }
+    }
+    .input {
+      width: 560px;
+      padding-top: 20px;
+    }
+    div#covers.input {
+      width: 560px;
+      max-height: 132px;
+      overflow: hidden;
+      padding-top: 20px;
+      margin-left: -80px;
+      text-align: center;
+    }
+    input[type=text],
+    input[type=file] {
+      width: 350px;
+      font-size: 1.125em;
+      font-family: "Lucida Grande", Verdana, Geneva, Helvetica, Arial, sans-serif;
+      padding: 3px;
+      margin-left: 30px;
+    }
+    button[type=submit] {
+      font-size: 1.125em;
+      width: auto !important;
```

Comment:
Do we need this `!important`?

Reply:
Let me check :)

Reply:
No I dont think so, without it also works fine :)

Reply:
Removed it